### PR TITLE
Added Italian translations

### PR DESCRIPTION
**Pull request by Hack64**

### CODE REVIEW

**Comment by idoybh** [comment]
We had to revert type clock PR. It didn't compile for some reason. Can you please fix and reopen that? I am checking this one locally now

File: res/values-it/derpquest_strings.xml [hunk 1]
```diff
@@ -0,0 +1,796 @@
...
+    <string name="battery_light_low_color">Livello basso</string>
+    <string name="battery_light_medium_color">Livello medio</string>
+    <string name="battery_light_full_color">Quasi carico</string>
+    <string name="battery_light_reallyfull_color">Completamente carico</string>
+
+    <string name="notification_light_settings">Led di notifica</string>
+
+    <!-- Fingerprint authentication vibration -->
+    <string name="fprint_sucess_vib_title">Vibrazione autenticazione FOD</string>
+    <string name="fprint_sucess_vib_summary">Vibra quando l'impronta digitale viene riconosciuta</string>
```
Comment (idoybh) [inline res/values-it/derpquest_strings.xml]: add `\` before the `'`

File: res/values-it/derpquest_strings.xml [hunk 2]
```diff
@@ -0,0 +1,796 @@
...
+    <string name="double_tap_sleep_lockscreen_title">Doppio tocco schermata di blocco per spegnere lo schermo</string>
+    <string name="double_tap_sleep_lockscreen_summary">Tocca due volte in qualsiasi punto della lockscreen per spegnere lo schermo</string>
+
+    <!-- Disable power menu on secure lock screen -->
+    <string name="lockscreen_power_menu_disabled_title">Disabilita menù spegnimento quando il telefono è bloccato</string>
+    <string name="lockscreen_power_menu_disabled_summary">Disabilità il menù di spegnimento quando il telefono è bloccato e protetto da password.</string>
+
+    <!-- Battery Bar -->
+    <string name="tuner_keyguard_show_battery_bar_title">Mostra barra batteria</string>
+    <string name="tuner_keyguard_show_battery_bar_summary">Mostra barra batteria sull'always on display quando il dispositivo è in carica</string>
```
Comment (idoybh) [inline res/values-it/derpquest_strings.xml]: same here

File: res/values-it/derpquest_strings.xml [hunk 3]
```diff
@@ -0,0 +1,796 @@
...
+    <string name="status_bar_clock_style_center">Centro</string>
+    <string name="status_bar_clock_style_left">Sinistra</string>
+    <string name="status_bar_am_pm_title">AM/PM</string>
+    <string name="status_bar_am_pm_info">Ora a 24 ore</string>
+    <string name="status_bar_am_pm_normal">Normale</string>
+    <string name="status_bar_am_pm_small">Piccolo</string>
+    <string name="status_bar_am_pm_hidden">Nascosto (predefinito)</string>
+    <string name="clock_seconds_display">Mostra secondi</string>
+    <string name="clock_date_display">Data</string>
+    <string name="clock_date_display_summary">Mostra data personalizzata a sinistra dell'ora</string>
```
Comment (idoybh) [inline res/values-it/derpquest_strings.xml]: same

File: res/values-it/derpquest_strings.xml [hunk 4]
```diff
@@ -0,0 +1,796 @@
...
+    <string name="clock_date_display_normal">Dimensione font media</string>
+    <string name="clock_date_style">Stile data</string>
+    <string name="clock_date_style_normal">Normale</string>
+    <string name="clock_date_style_lowercase">Minuscolo</string>
+    <string name="clock_date_style_uppercase">Maiuscolo</string>
+    <string name="clock_date_format_custom">Formato java personalizzato</string>
+    <string name="clock_date_format">Formato data</string>
+    <string name="clock_date_string_edittext_title">Deve essere necessariamente in DateFormat es. MM/dd/yy</string>
+    <string name="clock_date_string_edittext_summary">Inserisci stringa</string>
+    <string name="clock_date_right">A destra dell'ora</string>
```
Comment (idoybh) [inline res/values-it/derpquest_strings.xml]: you get the idea. just find and add `\` before any `'`. Amend and force push to the same branch please

**Review by idoybh — CHANGES_REQUESTED** [review]

File: res/values-it/derpquest_strings.xml (hunk 4 above)
Comment (Hack64) [reply res/values-it/derpquest_strings.xml]: Sure, I'll fix it asap and sorry for the inconveniences

**Review by Hack64 — COMMENTED** [review]
Fixed issue with **'** character

**Comment by Hack64** [comment]
Also I checked the clock PR but everything seems fine to me, can't really tell what's the issue :/

**Comment by idoybh** [comment]
> Also I checked the clock PR but everything seems fine to me, can't really tell what's the issue :/

I think there are some Italian characters the compiler doesn't like. It gives no specific characters / lines though. Just fails to compile the file. Same goes for this PR. Think of any non-english char that can cause that?

**Comment by Hack64** [comment]
> > Also I checked the clock PR but everything seems fine to me, can't really tell what's the issue :/
> 
> I think there are some Italian characters the compiler doesn't like. It gives no specific characters / lines though. Just fails to compile the file. Same goes for this PR. Think of any non-english char that ca cause that?

Is it possible that spaces cause issue? I basically copied the german xml and translated the strings, nothing else. Also there isn't any special char like è é ò and so on in the clock PR. I don't really know what could be, even Andrin said it was okay.

Is there anything basic that i'm missing? It should be UTF-8

**Comment by idoybh** [comment]
> > > Also I checked the clock PR but everything seems fine to me, can't really tell what's the issue :/
> > 
> > 
> > I think there are some Italian characters the compiler doesn't like. It gives no specific characters / lines though. Just fails to compile the file. Same goes for this PR. Think of any non-english char that ca cause that?
> 
> Is it possible that spaces cause issue? I basically copied the german xml and translated the strings, nothing else. Also there isn't any special char like è é ò and so on in the clock PR. I don't really know what could be, even Andrin said it was okay.
> 
> Is there anything basic that i'm missing? It should be UTF-8

Should be utf-8, yes. Maybe line endings. Will check now

**Comment by idoybh** [comment]
It wasn't line endings. I wish the compiler was more clear about this :(

**Comment by Hack64** [comment]
> It wasn't line endings. I wish the compiler was more clear about this :(

I wish it was too, I'm so sorry about this, but if we can't find a solution then whatever, i'm happy since i tried to contribute nonetheless :)

File: res/values-it/derpquest_strings.xml [hunk 5]
```diff
@@ -0,0 +1,796 @@
...
+    <string name="qs_always_show_settings_title">Mostra sempre icona impostazioni</string>
+    <string name="qs_always_show_settings_summary">Hides drag handle</string>
+
+    <!-- QS footer text -->
+    <string name="derp_footer_text_show_title">Mostra testo sul footer dei QS</string>
+    <string name="derp_footer_text_string_title">Personalizza testo footer</string>
+    <string name="derp_footer_text_string_summary">Lascialo vuote per il testo predefinito #DerpFest</string>
+
+    <!-- Battery mode -->
+    <string name="qs_hide_battery_title">Nascondi icona batteria<string>
```
Comment (idoybh) [inline res/values-it/derpquest_strings.xml:416]: bingo

**Review by idoybh — CHANGES_REQUESTED** [review]

File: res/values-it/derpquest_strings.xml (hunk 5 above)
Comment (idoybh) [reply res/values-it/derpquest_strings.xml:416]:
```suggestion
    <string name="qs_hide_battery_title">Nascondi icona batteria</string>

```

Comment (idoybh) [reply res/values-it/derpquest_strings.xml:416]: will merge and amend

**Pull request merged**